packages/commercetools/api-client/src/api/categorySearch/index.ts

```diff
@@ -0,0 +1,33 @@
+import gql from 'graphql-tag';
+import defaultQuery from './defaultQuery';
+import { CategorySearchResult } from '../../types/GraphQL';
+import { buildCategoryFilter } from '../../helpers/search';
+import ApolloClient from 'apollo-client';
+import { CustomQuery } from '@vue-storefront/core';
+
+export interface CategoryData {
+  categorySearch: CategorySearchResult;
+}
+
+const categorySearch = async (context, params, customQuery?: CustomQuery) => {
```

Comment:
Can we also add typings to params? Thanks to that we would know what properties are accessible